bentoml/_internal/bento/bento.py

```diff
@@ -302,9 +302,111 @@ def save(
 
         return self
 
-    def export(self, path: str):
-        out_fs = fs.open_fs(path, create=True, writeable=True)
+    Format: TypeAlias = t.Literal["bento", "gz", "xz", "bz2", "tar", "zip", "folder"]
+    Destination: TypeAlias = t.Literal["fs", "s3", "ftp"]
+    hasSchemeRe = re.compile(r".*[^\\]://")
+
+    def export(
+        self,
+        path: str,
+        dest: t.Optional[Destination] = None,
+        output_format: t.Optional[Format] = None,
+        *,
+        user: t.Optional[str] = None,
```

Comment:
rename to `ftp_user` and `ftp_passwd` to be more clear to users?

Reply:
This is actually also applicable elsewhere so I'm moving these to work more generally.